src/cpp/src/visual_language/nanollava/classes.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "visual_language/nanollava/classes.hpp"
+#include "visual_language/clip.hpp"
+#include "utils.hpp"
+
+namespace ov::genai {
+
+const std::string NATIVE_TAG = "<image>\n";
+const int64_t IMAGE_TOKEN_ID = -200;
```

Comment:
```suggestion
const int64_t IMAGE_PLACEHOLDER = -200;
```
It was confusing to see `tokenize_without_image_tag()` extracting `<image>` just to insert `IMAGE_TOKEN_ID`. Without noticing its value is negative it seemed like reverting. Keep `IMAGE_TOKEN_ID` if it's the name used in python implementation.

Would it be possible to search for `<image>` corresponding token id? I guess it was implemented the way you did it originally, just curious

Reply:
The value is fine. The name of the var was confusing

Reply:
Agree, renamed.